File: Java Solutions/Eligibility.java
import java.util.Scanner;

public class Eligibility {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int cases = sc.nextInt();
		sc.nextLine();
		
		while(cases != 0) {
			
			String input = sc.nextLine();
			String arr[] = input.split("\\W");
			
			int y1 = Integer.parseInt(arr[1]);
			int y2 = Integer.parseInt(arr[4]);
			int courses = Integer.parseInt(arr[7]);
			
				if(y1 >= 2010 || y2 >= 1991) {
					System.out.println(arr[0] + " eligible");
				} else if(courses > 40) {
					System.out.println(arr[0] + " ineligible");
				} else {
					System.out.println(arr[0] + " coach petitions");
				}
			cases--;
		}
	}
}

File: Kattis-Problems/Python-Solutions/Eligibility.py
a = int(input())
for i in range(a):
    arr = list(input().split())
    arrTwo = list(arr[1].split("/"))
    arrThree = list(arr[2].split("/"))

    if int(arrTwo[0]) >= 2010 or int(arrThree[0]) >= 1991:
        print(arr[0] + ' eligible')
    elif int(arr[3]) > 40:
        print(arr[0] + ' ineligible')
    else:
        print(arr[0] + ' coach petitions')